### PR TITLE
fix hrefs values

### PR DESCRIPTION
**Pull request by JustinCast**
# Fix incorrect href values
Change hrefs values from jungle.bloks.io to jungle3.bloks.io

## How this should be tested
- Type `make run` in the terminal
- Go to `localhost:3000` and see the magic﻿

### CODE REVIEW

File: webapp/src/components/MapPopup/index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function MapPopup({ id, info, account }) {
       <div>
         Account:{' '}
         <a
-          href={`https://jungle.bloks.io/account/${account}`}
+          href={`https://jungle3.bloks.io/account/${account}`}
```
Comment (xavier506) [inline webapp/src/components/MapPopup/index.js:35]: Should we set this as an env var?

**Review by xavier506 — APPROVED** [review]

File: webapp/src/components/MapPopup/index.js (hunk 1 above)
Comment (JustinCast) [reply webapp/src/components/MapPopup/index.js]: Fixed in the last commit!.

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by JeanVegaD — APPROVED** [review]

File: webapp/src/components/RicardianContract/RicardianContract.js [hunk 2]
```diff
@@ -43,8 +43,8 @@ const RicardianContract = ({ name, hash, abi, action }) => {
     )
 
     const mainTitle = '# Ricardian contract'
-    const nameSection = `**Name:** [${name}](https://jungle.bloks.io/account/${name}?loadContract=true&tab=Actions)`
-    const hashSection = `**Hash:** [${hash}](https://jungle.bloks.io/account/${name}?loadContract=true&tab=ABI)`
+    const nameSection = `**Name:** [${name}](${process.env.REACT_APP_BLOCK_EXPLORER_URL}account/${name}?loadContract=true&tab=Actions)`
```
Comment (adriexnet) [inline webapp/src/components/RicardianContract/RicardianContract.js:46]: Same issue here, please use src/config and then import the variable from there

**Review by adriexnet — CHANGES_REQUESTED** [review]

**Review by adriexnet — APPROVED** [review]
LGTM

**Pull request merged**